cmd/src/main.go

```diff
@@ -11,6 +11,14 @@ import (
 	"strings"
 )
 
+const (
+	AccessTokenEnvVar = "SRC_ACCESS_TOKEN"
+	ConfigEnvVar      = "SRC_CONFIG"
+	ConfigFilename    = "src-config.json"
+	DefaultEndpoint   = "https://sourcegraph.com"
+	EndpointEnvVar    = "SRC_ENDPOINT"
```

Comment:
Constants like these shouldn't be exported, they're not imported by other code external to this package.